js/apps/account-ui/src/personal-info/PersonalInfo.tsx

```diff
@@ -38,6 +39,7 @@ export const fieldName = (name: string) =>
 
 const PersonalInfo = () => {
   const { t } = useTranslation();
+  const context = useKeycloak();
```

Comment:
Can we name this something other than `context`?
```suggestion
  const keycloak = useKeycloak();
```